modules/ROOT/pages/java/java-invoke-method.adoc

```diff
@@ -3,16 +3,18 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-You can invoke Java methods by using operations in the Java module or in
-a DataWeave expression.
+You can invoke Java methods (either instance or static) through the `invoke` and `invoke static` operations in the Java module.
 
 == Invoke Instance Methods
 
 This example shows how to invoke instance methods: 
 
-. You can the `new` operation (`<java:new/>`) from the Java module to create the instance on which
-the method is invoked. 
-. Use the `invoke` operation from the Java module to invoke a method (`init()`) that does not take any arguments.
+. First you need an instanced object to call one of its methods. You can use the xref:java/java-create-instance.adoc[`new`
+operation] from the Java module to create an instance on which a method is later
+invoked. In the example, we will create an instance of the class _TaxCalculator_ and save it
```

Comment:
we can't provide examples using classes of classes that we are not proving

Reply:
added java code for used classes